examples/multigrid/multigrid.c

```diff
@@ -274,12 +274,12 @@ int main(int argc, char **argv) {
   // Create the restriction/interpolation Q-function
   CeedQFunctionCreateInterior(ceed, 1, bpOptions[bpChoice].ident,
                               bpOptions[bpChoice].identfname, &qf_restrict);
-  CeedQFunctionAddInput(qf_restrict, "uin", vscale, CEED_EVAL_INTERP);
-  CeedQFunctionAddOutput(qf_restrict, "uout", vscale, CEED_EVAL_NONE);
+  CeedQFunctionAddInput(qf_restrict, "uin", vscale, CEED_EVAL_NONE);
+  CeedQFunctionAddOutput(qf_restrict, "uout", vscale, CEED_EVAL_INTERP);
   CeedQFunctionCreateInterior(ceed, 1, bpOptions[bpChoice].ident,
                               bpOptions[bpChoice].identfname, &qf_prolong);
-  CeedQFunctionAddInput(qf_prolong, "uin", vscale, CEED_EVAL_NONE);
-  CeedQFunctionAddOutput(qf_prolong, "uout", vscale, CEED_EVAL_INTERP);
+  CeedQFunctionAddInput(qf_prolong, "uin", vscale, CEED_EVAL_INTERP);
```

Comment:
I was actually using prolongation as the transpose of restriction rather than restriction as the transpose of prolongation - but this change is doing worse in some cases.

Reply:
Gotta test the interpolation-is-embedding property.

If your fine-scale solution has a component in direction `P_n` (the Legendre polynomial of highest degree) and you sample it on a smaller number of points, you get full-amplitude aliasing of a component that we don't want to represent at all on the coarse grid. Although it may not be the issue, the numbers you've been reporting are consistent with what happens when the grid transfer operators alias excessively.

Reply:
For example - on the 20x20x20 box above with p=7 I'm now up to 85 iterations instead of 24 even though it seems like
```
G_Fine^T B_Identity D_Identity B_(Coarse to Fine Interpolation) G_Coarse
```
is better for interpolation than
```
G_Fine^T B_(Fine to Coarse Interpolation)^T D_Identity B_Identity G_Coarse
```

Reply:
I think I was getting some alising with my more aggressive coarsening strategy. I'm trying without this change and a slower coarsening.